modules/product/tests/src/FunctionalJavascript/ProductAttributeJavascriptTest.php

```diff
@@ -73,14 +73,12 @@ public function testProductAttributeValues() {
     $this->assertSession()->pageTextContains('Updated the Color product attribute.');
 
     $attribute = ProductAttribute::load('color');
-    $attribute_values = [];
-    /** @var \Drupal\commerce_product\Entity\ProductAttributeValueInterface $value */
-    foreach ($attribute->getValues() as $value) {
-      $attribute_values[] = $value->label();
-    }
-    $this->assertTrue($attribute_values[0] == 'Cyan');
-    $this->assertTrue($attribute_values[1] == 'Magenta');
-    $this->assertTrue($attribute_values[2] == 'Cornflower Blue');
+    /** @var \Drupal\commerce_product\Entity\ProductAttributeValueInterface[] $attribute_values */
+    $attribute_values = $attribute->getValues();
```

Comment:
I'm stupid. This is keyed by entity ID. Above I properly foreach the results to re-key by weight value.